test/unit/model/reference/index.test.ts

```diff
@@ -11,5 +12,19 @@ describe('Reference class', () => {
     expect(ref.authors).toEqual([]);
     expect(ref.type).toEqual('journal');
     expect(ref.id).toEqual('unique_id');
+    expect(ref.referenceInfo).toBeInstanceOf(JournalReference);
```

Comment:
Just a thought, should we just mock these classes for this and assert the constructor was called with empy args & `expect(ref.referenceInfo).toBeInstanceOf(JournalReference);` ?